utils/fw_tunnel_updater.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import getpass
+import sys
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+from rffe_nuttx_lib import RFFEFWUpdate
+from rffe_tunnel_lib import SSHTunnel
+
+
+def load_rack_config(file_path):
+    with open(file_path) as f:
+        return json.load(f)
+
+def get_getway(rack_id):
+    if rack_id == "TL":
+        return "ia-20rabpmtl-co-iocsrv.lnls-sirius.com.br"
+    else:
+        return f"ia-{rack_id}rabpm-co-iocsrv.lnls-sirius.com.br"
```

Comment:
Suggestion: Get the hostnames from the JSON file instead of generating these here.